templates/index.html

```diff
@@ -0,0 +1,4 @@
+<h1>Headlines</h1>
```

Comment:
I think this is missing the `<html>`, `<head>`, `<body>`, tags to be "correct" html